source/server/http/admin.cc

```diff
@@ -279,11 +290,42 @@ Http::Code AdminImpl::handlerStats(const std::string&, Buffer::Instance& respons
     all_stats.emplace(gauge->name(), gauge->value());
   }
 
-  for (auto stat : all_stats) {
-    response.add(fmt::format("{}: {}\n", stat.first, stat.second));
+  if (params.size() == 0) {
+    // No Arguments so use the standard
+    for (auto stat : all_stats) {
+      response.add(fmt::format("{}: {}\n", stat.first, stat.second));
+    }
+  } else {
+    std::string format = params.begin()->first;
+    std::string formatvalue = params.begin()->second;
```

Comment:
Please use snake case for all vars (e.g., "format_value", "stats_array" below, etc.). Won't comment on each one.

Reply:
will change.. this habit from my lot of java coding :-)